apps/site/assets/ts/stop/__tests__/DepartureListTest.tsx

```diff
@@ -83,7 +87,11 @@ describe("DepartureList", () => {
         alerts={[]}
       />
     );
-    expect(screen.getByText(predictionTime.toString())).toBeDefined();
+    expect(container.querySelector("time")).toBeDefined();
+    const datetimes = Array.from(container.querySelectorAll("time")).map(el =>
+      el.getAttribute("datetime")
+    );
+    expect(datetimes).toContain(predictionTime.toISOString());
```

Comment:
Same here.  `container` should only be used if all other avenues failed.  Such as checking if a specific color gets applied to a background.